src/utils/network-monitor.ts

```diff
@@ -569,6 +569,24 @@ export class NetworkMonitor {
     process.on('exit', this.exitHandler)
   }
 
+  async checkWalletBalances(address: string, networks: number[]): Promise<void> {
```

Comment:
@alexanderattar So the above object you have is for creating txs for our protocol. The min gas i have in this function is to make sure an operator has enough balance to be able to submit a tx. 

A few times, i have had an operator wallet have zero / no balance when booting up. This then causes all estimate gas calls to fail because my wallet has no balance to submit a tx. This continues for all jobs in my tx array. So i created this new function that will check the balance of the operator wallet on boot. I can add comments to make this clearer.

Reply:
Okay cool thanks for explaining. Yeah I think just adding those comments will be helpful to remember what's happening in the future if we need to modify this stuff again 👍